tests/test_files_directory/nested_directory/build_test_merged_file.json

```diff
@@ -1,7 +1,3 @@
 {
-    "Hello": "There",
-    "UhOh": "This",
-    "OneIs": "Nested",
-    "UhOh2": "This",
-    "OneIs2": "NestedAlso"
+    "Hello": "There"
```

Comment:
This is a really badly written test that actually writes to a file that it pulls data from, hence failing all future unit tests. I have to remember to remove these added lines after running unit tests every time. It's something I/we should fix when we address tests

Reply:
The corresponding file suffixed with _ORIGINAL specifies how the file should look when in it's original form FYI